examples/hawkbit-device-simulator/src/main/java/org/eclipse/hawkbit/simulator/ui/GenerateDialog.java

```diff
@@ -231,4 +185,64 @@ public boolean equals(final Object obj) {// NOSONAR - as this is generated
         void okButton(final String namePrefix, final String tenant, final int amount, final int pollDelay,
                 final URL basePollURL, final String gatewayToken, final Protocol protocol);
     }
+
+    private OptionGroup createProtocolGroup() {
+
+        final OptionGroup protocolGroup = new OptionGroup("Simulated Device Protocol");
+        protocolGroup.addItem(Protocol.DMF_AMQP);
+        protocolGroup.addItem(Protocol.DDI_HTTP);
+        protocolGroup.setItemCaption(Protocol.DMF_AMQP, "Device Management Federation API (AMQP push)");
+        protocolGroup.setItemCaption(Protocol.DDI_HTTP, "Direct Device Interface (HTTP poll)");
+        protocolGroup.setNullSelectionAllowed(false);
+        protocolGroup.select(Protocol.DMF_AMQP);
+        protocolGroup.addValueChangeListener(event -> {
+            pollDelayTextField.setVisible(!pollDelayTextField.isVisible());
```

Comment:
Is this right only to switch the visibilty? 

The code before has a different behavior...
`
     if (event.getProperty().getValue().equals(Protocol.DDI_HTTP)) {
                    tf4.setVisible(true);
                    tf5.setVisible(true);
                    tf6.setVisible(true);
                } else {
                    tf4.setVisible(false);
                    tf5.setVisible(false);
                    tf6.setVisible(false);
                }
`

in my opinion something like that is right:
` tf4.setVisible(event.getProperty().getValue().equals(Protocol.DDI_HTTP));`
etc...